problemsets-mat223/modules/module14.tex

```diff
@@ -122,13 +129,13 @@
 	Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{2x+y\\-x+\tfrac{1}{2}y}$.
 	Find $\det(\mathcal T)$.
 
-	XXX Finish (this is the example from earlier)
+	Refer to the example from earlier, $\mathcal T$ is orientation preserving. Therefore, $\det(\mathcal T)=2$.
```

Comment:
This need a figure to explain. You can put an `XXX Figure` here.

Reply:
Prof, this does not need a graph. You will have a figure in the previous example

Reply:
Wow Toronto people are so amazing that I am not able to sleep anymore hahaha. Let's me finish the latex